src/language-css/parser-postcss.js

```diff
@@ -85,6 +85,7 @@ function parseValueNode(valueNode, options) {
         (!hasStringOrFunction(groupList) && !isSCSSVariable(groupList[0]))
       ) {
         const stringifiedContent = stringifyNode({
+          type: node.group.type,
```

Comment:
Would it work if we pass node.group directly?

Reply:
Maybe no.
I think we need to omit open and close.
Maybe `{ ...node.group, open: null, close: null }` works.
I'll see it later.

Reply:
That won't be necessary.